config/initializers/assets.rb

```diff
@@ -1,3 +1,5 @@
-if Rails.application.config.should_show_easy_login
-  Rails.application.config.assets.precompile += %w( admin.css public.css easy_sign_in.js )
-end
+Rails.application.config.assets.precompile += %w(
+  admin.css
+  easy_sign_in.js
```

Comment:
@Ruxton I'm not really following you. What do you mean buy it will sit on the server and be available if required? Won't it be included in the js end served up to every client?

Also what do you mean by the correct way to do asset compilation is on your local machine? Hasn't tfg_cap always done the asset compilation on the server? Doing asset compilation on my local machine and pushing the result sounds like a horrible deploy process to me.